index.js

```diff
@@ -0,0 +1,59 @@
+//Problem 1
+
+// - ### Sum of all from 1 to n
+
+// Write a function that takes in a number as input and recursively finds the sum of all numbers up to and including that number.
+
+// ```js
+// input: 6
+// output: 21
+
+// //21 = 6 + 5 + 4 + 3 + 2 + 1
+// ```
+
+const sumOfNum = (num,sum = 0) => {
+    if(num === 0) return sum;
+    sum += num
+    num--
+    return sumOfNum(num,sum)
+}
+// console.log(sumOfNum(6))
+
+//Problem 2
+
+// - ### Multiply array
+
+// Write a function called `multArr` that takes in an array of numbers as an argument and recursively multiplies together all of the values in the array.
+
+// ```js
+// multArr([2, 3, 5]); // returns 30
+// multArr([5, 5, 1, 2]); //returns 50
+// ```
+
+const multArr = (arr, product = 1) => {
```

Comment:
You can completely remove the necessity for having product as an argument if you set your return on line 14 to be 1.